### ISSUE TITLE
Add `gopass git remote list` to show remotes

### ISSUE THREAD

**Issue by HectorMalot** [gopasspw/gopass#2088]
### Feature request
I'd like to propose adding `gopass git remote list` next to  `gopass git remote add` and `gopass git remote remove`. There currently is no convenient way to list your remotes (e.g. when you have multiple remotes, or want to confirm the name for a removal action.

### Context
I've recently migrated from pass to gopass. I'm using a multiple remotes and they're not named origin. I find myself typing `gopass git remote` often to look at which remote I want to push to, but this feature currently does not exists.

I have some experience with Go and am happy to help implement this, but wanted to confirm first whether you would accept a pull request for this functionality.

**Comment by dominikschulz** [comment]
We used to just pass through any invocations of `gopass git` to a git instance cwd'd to the right store but stopped that when we did experiment with custom storage (RCS) backends. Since this experiement didn't prove to be useful and has been stopped we could consider just bringing back forwarding commands to `git`.

That would remove some amount of code we'd otherwise have to maintain and bring us closer to `pass` (again).
